starter-frontend/vite.config.js

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react"
+
+export default defineConfig(() => {
+    return {
+        // GITHUB PAGES SPECIFIC CONFIG
+        // TODO: REMOVE WHEN DEPLOYING TO AWS
+        base: '/cool-husky-games/',
+        build: {
+            outDir: 'build',
+        },
```

Comment:
## Notes ❗ 
Vite seems like a cool technology! I saw in a vite tutorial that the following code block is what you added to the template. Is this supposed to be removed when migrating to AWS or just the base /cool-husky-games/?
'
        base: '/cool-husky-games/',
        build: {
            outDir: 'build',
        },
'

Reply:
It honestly depends on how you deploy to AWS, I'd recommend removing the entire code block of the `{base: '...', build: {...}}` config.